### PR TITLE
Added the minecart's flying velocity expression

### PR DESCRIPTION
**Pull request by DeltaRays**
### Description

---
**Target Minecraft Versions:** any 
**Requirements:** none 
**Related Issues:** none

### CODE REVIEW

**Comment by ShaneBeee** [comment]
I feel like maybe rather than having an expression for every aspect of a single entity, this one, 
https://github.com/SkriptLang/Skript/pull/3219,
and https://github.com/SkriptLang/Skript/pull/3218
could be merged into 1 expression

**Comment by DeltaRays** [comment]
Okay, I'll do that

**Comment by DeltaRays** [comment]
How about one for the max speed and one for both flying and derailed?

File: src/main/java/ch/njol/skript/expressions/ExprMinecartFlyingVelocity.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+	@Nullable
+	@Override
+	public Vector convert(Entity entity) {
+		return entity instanceof Minecart ? ((Minecart) entity).getFlyingVelocityMod() : null;
+	}
+	
+	@Nullable
+	@Override
+	public Class<?>[] acceptChange(ChangeMode mode) {
+		return (mode == ChangeMode.SET || mode == ChangeMode.ADD || mode == ChangeMode.REMOVE) ? CollectionUtils.array(Vector.class) : null;
```
Comment (Pikachu920) [inline src/main/java/ch/njol/skript/expressions/ExprMinecartFlyingVelocity.java:56]: switch case would be good especially cause of #3216

File: src/main/java/ch/njol/skript/expressions/ExprMinecartFlyingVelocity.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+		if (delta != null) {
+			switch (mode) {
+				case SET:
+					getExpr().stream(e).forEach(entity -> {
+						if (entity instanceof Minecart)
+							((Minecart) entity).setFlyingVelocityMod((Vector) delta[0]);
+					});
+					break;
+				case ADD:
+					getExpr().stream(e).forEach(entity -> {
```
Comment (Pikachu920) [inline src/main/java/ch/njol/skript/expressions/ExprMinecartFlyingVelocity.java:70]: for loops here, pleaseeee

Comment (DeltaRays) [reply src/main/java/ch/njol/skript/expressions/ExprMinecartFlyingVelocity.java:70]: Sorryyyyyy

**Comment by DeltaRays** [comment]
Closing in favour of #3219

**Pull request closed without merging**